flag.go

```diff
@@ -347,8 +347,11 @@ func stringifyFlag(f Flag) string {
 
 	defaultValueString := ""
 
-	if s := df.GetDefaultText(); s != "" {
-		defaultValueString = fmt.Sprintf(formatDefault("%s"), s)
+	// dont print default text for required flags
+	if rf, ok := f.(RequiredFlag); !ok || !rf.IsRequired() {
```

Comment:
Maybe this test could be moved in a function of `f`

```suggestion
	if rf, ok := f.(RequiredFlag); !ok || !rf.IsRequired() {
```

Reply:
I dont understand. Can you elaborate ?